frontend/src/pages/CollaborationReportForm/Pages/supportingInformation.js

```diff
@@ -1,20 +1,260 @@
+            id="yesDataUsed"
+            name="hasDataUsed"
+            value="true"
+            label="Yes"
+            className="smart-hub--report-checkbox"
+            inputRef={register({ required: 'Select one' })}
+            required
+          />
+          <Radio
+            id="noDataUsed"
+            name="hasDataUsed"
+            value="false"
+            label="No"
+            className="smart-hub--report-checkbox"
+            inputRef={register({ required: 'Select one' })}
+          />
+        </FormItem>
+      </Fieldset>
+      {showDataUsedOptions && (
+        <Fieldset className="smart-hub--report-legend">
+          <FormItem
+            label="What data did you collect, use, and/or share during this activity?"
+            name="dataUsed"
+            required
+          >
+            <MultiSelect
+              name="dataUsed"
+              control={control}
+              options={dataUsedOptions}
+              simple={false}
+              labelProperty="label"
+              valueProperty="value"
+              required
+            />
+          </FormItem>
+        </Fieldset>
+      )}
+      {showOtherDataUsed && (
+        <Fieldset className="smart-hub--report-legend">
+          <FormItem
+            label="Others data collected"
+            name="otherDataUsed"
+            fieldSetWrapper
+            required
+          >
+            <TextInput
+              id="otherDataUsed"
+              name="otherDataUsed"
+              type="text"
+              inputRef={register({
+                required: 'Enter other data',
+              })}
+            />
+          </FormItem>
+        </Fieldset>
+      )}
+      <Fieldset className="smart-hub--report-legend">
+        <FormItem
+          label="Does the content of this activity help recipients in your region support their goals?"
+          name="hasGoals"
+          required
+        >
+          <Radio
+            id="yesGoals"
+            name="hasGoals"
+            value="true"
+            label="Yes"
+            className="smart-hub--report-checkbox"
+            inputRef={register({ required: 'Select one' })}
+            required
+          />
+          <Radio
+            id="noGoals"
+            name="hasGoals"
+            value="false"
+            label="No"
+            className="smart-hub--report-checkbox"
+            inputRef={register({ required: 'Select one' })}
+          />
+        </FormItem>
+      </Fieldset>
+      {showGoalsOptions && (
+        <Fieldset className="smart-hub--report-legend">
+          <FormItem
+            label="Select the goals that this activity supports"
+            name="goals"
+            required
+          >
+            <MultiSelect
+              name="goals"
+              control={control}
+              options={goalsOptions}
+              simple={false}
+              labelProperty="label"
+              valueProperty="value"
+              required
+            />
```

Comment:
I misremembered, I think in the past I've gotten the front end and backend to agree using labelProperty and valueProperty but not using our MultiSelect component. I just used react-select, imported our styles object, and used the built in ways that library has to use custom label and value props. There's probably examples on the sessionSummary page.

 I think in this case if that sounds like too gnarly, I would go with what you have and just handle the transformation on the frontend. Just note right now there are multiple save functions in CollaborationReportForm/index.js and you'll have to make sure to transform it on the way out and the way in a few different places. Same goes for data used.